contracts/core/lib/auction-price-libraries/LinearAuctionPriceCurve.sol

```diff
@@ -97,6 +100,12 @@ contract LinearAuctionPriceCurve {
         uint256 priceNumerator = _auctionParameters.auctionPivotPrice;
         uint256 currentPriceDenominator = priceDenominator;
 
+        // Determine the auctionStartPrice based on if it should be sef-defined
```

Comment:
We should add a warning comment about the implications of this for future coders, as this could easily be a source of bugs.

Reply:
Also `self-defined`